docs/content/preview/yugabyte-voyager/reference/data-migration/export-data.md

```diff
@@ -8,64 +8,82 @@ menu:
     identifier: voyager-export-data
     parent: data-migration
     weight: 20
+aliases:
+  - /preview/yugabyte-voyager/reference/fall-forward/fall-forward-synchronize/
 type: docs
+rightNav:
+  hideH4: true
 ---
 
+The following page describes the following export commands:
 
-This page describes the following export commands:
-
-- [export data](#export-data)
+- [export data](#export-data) (and export data from source)
 - [export data status](#export-data-status)
+- [get data-migration-report](#get-data-migration-report)
+- [export data from target](#export-data-from-target)
 
-## export data
+### export data
```

Comment:
Why bump heading for commands down to 3? Worthy of 2...

Reply:
I did that for right nav purpose. On second thoughts, it looks like I can bump it up as the syntax, arguments and example showing up may be useful. Ill change it.